### ISSUE TITLE
weird characters show up with git-radar on mac os x

### ISSUE THREAD

**Issue by thomasdziedzic** [michaeldfallen/git-prompt#85]
This only happens in 1 of my repositories, the other ones work just fine.
It used to work, but I don't know what has changed since then.
Let me know how I can help figure this out, thanks!

```
$ git-radar --bash --fetch
 git:(master) 1��
```

Here is a screenshot:

![http://i.imgur.com/oKpSY06.png](http://i.imgur.com/oKpSY06.png)

**Comment by hallzy** [comment]
Do you have anything stashed? There was something added that does something like that to track stashes, except that the character is supposed to be 3 horizontal lines.

If it is a stash not sure how to fix it since I don't use osx.

**Comment by thomasdziedzic** [comment]
yes, I had a stash, after clearing it with `git stash clear`, it worked!

**Comment by michaeldfallen** [comment]
That's a little worrying. The stash icon is unicode ≡ (indentical to). We had presumed most terminals would support it. What terminal emulator do you use and which version of bash?

**Comment by thomasdziedzic** [comment]
I use iTerm (with tmux sometimes).

```
$ bash --version
GNU bash, version 4.3.42(1)-release (x86_64-apple-darwin14.5.0)
Copyright (C) 2013 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
```

**Comment by ctrlrsf** [comment]
Seeing this as well in Terminal, iTerm, and xterm running under XQuartz. Suspect it started after installing latest OS X update (10.11.1) this morning. Release notes mention: Adds over 150 new emoji characters with full Unicode 7.0 and 8.0 support.

https://support.apple.com/en-us/HT205360

![image](https://cloud.githubusercontent.com/assets/6004738/10828636/e9dba250-7e4b-11e5-80d7-440aff94b8c6.png)

**Comment by ctrlrsf** [comment]
Just want to confirm `git stash clear` gets rid of the ??'s, and running `git stash` after making local changes brings them back.

**Comment by c5f** [comment]
I fixed this on my local system by calling `printf` on the congruency/equivalency character in `radar-base.sh:542`:

``` shell
printf $PRINT_F_OPTION "$number_stashes$COLOR_STASH`printf ≡`$RESET_COLOR_STASH"
```

Original:

``` shell
printf $PRINT_F_OPTION "$number_stashes$COLOR_STASH≡$RESET_COLOR_STASH"
```

I'm still running Yosemite so it is not an El Capitan-specific issue.  This is a dumb hacky fix that I pulled from surrounding code because I didn't feel like learning the bash way to do it - please don't judgy. :beers:

This leads me to believe that the unicode character is conflicting with either color escape sequences or is being gobbled up into the environment variable.  Colors still work on local changes which follow the stash changes, so `$RESET_COLOR_STASH` is working properly.

---

This change fixes stash symbol rendering for iTerm2.app build 2.1.4 and Terminal.app Version 2.5.3 (343.7) on OS X 10.10.5 (14F1021).

**Comment by michaeldfallen** [comment]
This should be fixed now thanks to @matthillman pull #90. Closing.
